composer.json

```diff
@@ -22,15 +22,14 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
```

Comment:
2.* might be 2.1 as well. Guess bumping up requirements just because it's new LTS doesn't make sense. I'm not sure but guess this bundle still can work with symfony 2.3. Needs checking